File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L4/01/L4.ts
import { readlineSync } from 'readline-sync';

function main(): void {
  let nomor: number = 0;
  console.log('Enter an integer: ');
  
  nomor = Number(readlineSync.prompt());
  
  let sisa: number = 0;
  while (nomor !== 0) {
    sisa = nomor % 10;
    console.log(sisa);
    nomor = Math.floor(nomor / 10);
  }
  console.log();
}

main();

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-05/plagiarized/L4/01/L4.kt
package IR

import java.util.*

object L4 {
    @JvmStatic
    fun main(args: Array<String>) {
        var nomor = 0
        print("Enter an integer: ")

        val input = Scanner(System.`in`)

        nomor = input.nextInt()

        var sisa = 0
        while (nomor != 0) {
            sisa = nomor % 10
            print(sisa)
            nomor = nomor / 10
        }
        println()
    }
}

